### PR TITLE
Replace Entity Delete with active flag.

### LINKED ISSUE #59: Replace Entity delete with is_active flag
Allow the Entity active state to be set at the top level.

Remove the ability to Delete an Entity.

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**
Prevent deletion of entities but allow the user to make them inactive.
 - Closes #59.

### CODE REVIEW

File: spec/form_objects/entity_entry_spec.rb [hunk 1]
```diff
@@ -4,7 +4,17 @@
   subject { EntityEntry.new }
 
   describe 'validations' do
+    it { is_expected.to validate_presence_of(:is_active) }
     it { is_expected.to validate_presence_of(:name) }
     it { is_expected.to validate_presence_of(:reference) }
   end
+
+  describe '.build_from_entity' do
```
Comment (michaelachrisco) [inline spec/form_objects/entity_entry_spec.rb]: Were you going to add to these method specs?

**Comment by josephbridgwaterrowe** [comment]
One of these days :)

**Comment by michaelachrisco** [comment]
Does that mean that you want to push these tests to a later pull request?

**Comment by josephbridgwaterrowe** [comment]
Yes :)

**Comment by josephbridgwaterrowe** [comment]
@michaelachrisco, tests added... please review.

**Comment by michaelachrisco** [comment]
Looks good. I'll merge as soon as travis passes.

**Pull request merged**